changelogs/changelog.yaml

```diff
@@ -14,7 +14,7 @@ releases:
     release_date: '2024-05-07'
   1.1.0:
     changes:
-      major_changes:
+      minor_changes:
       - Added module vm_list_group_by_clusters
```

Comment:
I'm not sure if it's acceptable to alter the changelog after the release. Not 100% sure, though.

cc @Andersson007

Reply:
Thank I updated that file as well

Reply:
CHANGELOG.rst is likely autogenerated, you can use `antsibull-changelog generate` to update it from `changelogs/changelog.yaml`.

It's best to not manually edit the output file since antsibull-changelog will overwrite it for the next release anyway. Better only change the YAML file (either directly or indirectly through `antsibull-changelog release`) and let antsibull-changelog update the output files.

(You can also enable MarkDown output for antsibull-changelog BTW.)

Reply:
Thanks @felixfontein I reverted the manual change and ran `antsibull-changelog generate` and pushed it.